GoogleForCreators/web-stories-wp
# Updated library search component

## Context

Updated the library search component used in the element panel and the third-party content panel.

## User-facing changes

| Section | Before | After |
|-|-|-|
| Elements | ![Screen Shot 2021-02-25 at 18 24 21](https://user-images.githubusercontent.com/637548/109232873-f7230800-7796-11eb-8ed1-6228cf7c18e9.png) | ![Screen Shot 2021-02-25 at 18 19 25](https://user-images.githubusercontent.com/637548/109232885-fb4f2580-7796-11eb-9475-2e6846aa6f2d.png) |
| Searching | ![Screen Shot 2021-02-25 at 18 24 28](https://user-images.githubusercontent.com/637548/109232897-ff7b4300-7796-11eb-9bd8-941037b2d4f4.png) | ![Screen Shot 2021-02-25 at 18 19 20](https://user-images.githubusercontent.com/637548/109232904-01dd9d00-7797-11eb-9a32-3657eb5f864c.png) 
| Third-party | ![Screen Shot 2021-02-25 at 18 24 16](https://user-images.githubusercontent.com/637548/109232923-05712400-7797-11eb-9a86-eb20ca53d740.png) | ![Screen Shot 2021-02-25 at 18 19 33](https://user-images.githubusercontent.com/637548/109232936-0904ab00-7797-11eb-8862-70b0654da679.png) |

## Testing Instructions

### QA

This PR can be tested by following these steps:

1. Observe the search field in the local media element library
2. Search and observe functionality
3. Go to the third-party content and search for either type of external content
4. Observe functionality

### UAT




## Checklist




---

Fixes #6322
